packages/config/config/devices/0x0234/zba7140.json

```diff
@@ -0,0 +1,279 @@
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "templates/logic_group_template.json#enable_central_scene"
+		},
+		"2": {
+			"$import": "templates/logic_group_template.json#button_pressed_threshold_time"
+		},
+		"3": {
+			"$import": "templates/logic_group_template.json#button_held_threshold_time"
+		},
+		"4": {
+			"$import": "templates/logic_group_template.json#led_indicator_zba7140"
+		},
+		"5[0xff000000]": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "LED Indicator Red Color",
+			"valueSize": 4,
+			"defaultValue": 127
+		},
+		"5[0xff0000]": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "LED Indicator Green Color",
+			"valueSize": 4,
+			"defaultValue": 85
+		},
+		"5[0xff00]": {
+			"$import": "../templates/master_template.json#base_0-255_nounit",
+			"label": "LED Indicator Blue Color",
+			"valueSize": 4,
+			"defaultValue": 85
+		},
+		"5[0xff]": {
+			"$import": "templates/logic_group_template.json#multilevel_disabled_byte"
+		},
+		"6[0x01]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 2 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x02]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 3 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x04]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 4 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x08]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 5 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x10]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 6 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x20]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 7 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x40]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 8 Secure Message",
+			"valueSize": 2
+		},
+		"6[0x80]": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Association Group 9 Secure Message",
+			"valueSize": 2
+		},
+		"7": {
+			"$import": "templates/logic_group_template.json#pushbutton_functionality",
+			"label": "Pushbutton 1"
+		},
+		"10": {
+			"$import": "templates/logic_group_template.json#pushbutton_functionality",
+			"label": "Pushbutton 2"
+		},
+		"13": {
+			"$import": "templates/logic_group_template.json#pushbutton_functionality",
+			"label": "Pushbutton 3"
+		},
+		"16": {
+			"$import": "templates/logic_group_template.json#pushbutton_functionality",
+			"label": "Pushbutton 4"
+		},
+		"8[0xff000000]": {
+			"$import": "templates/logic_group_template.json#multilevel_enable",
+			"label": "Pushbutton 1 Single-Activation Enable"
```

Comment:
For consistency, lets make these: Button 1 - Single Press. No need to include Enable as the options will be Enable/Disable.